packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/task/TaskNotificationEventData.java

```diff
@@ -0,0 +1,36 @@
+package software.uncharted.terarium.hmiserver.models.task;
+
+import java.io.Serializable;
+import java.util.UUID;
+import lombok.Builder;
+import lombok.Value;
+import software.uncharted.terarium.hmiserver.annotations.TSModel;
+import software.uncharted.terarium.hmiserver.models.task.TaskRequest.TaskType;
+
+@Value
+@Builder
+@TSModel
+public class TaskNotificationEventData implements Serializable {
+	private UUID taskId;
+
+	private TaskType taskType;
+
+	private TaskStatus status;
+
+	private String message;
+
+	private String error;
+
+	private Object data;
```

Comment:
What is `data` here? If its just JSON data this can be a `JsonNode`

Reply:
Why are we not just sending the TaskResponse with all these fields as we did before? Why create a near identical class?

Reply:
I initially thought of creating an object with only necessary fields just for notification handling excluding actual output data. How large output field can get from the response object? If it's neglectable, I'll just use resp since this turned out very similar to the response object as you mentioned.

Reply:
I just updated to send the task response data.